### PR TITLE
added rest of copy

### LINKED ISSUE #12: Privacy Policy copy
## Description

We need to have a privacy policy on the site to describe how we are using cookies and what will happen with people data.

Here is a link to the privacy policy: https://docs.google.com/document/d/1HxiUJC9yihbY1DpUaBs3wVord6F2SjEYnzusys68Alw/edit?usp=sharing

## Plan

- [ ] Add copy to privacy policy page from Google Doc

## Acceptance Criteria

- [x] It is understood how we use peoples data and where it is stored
- [ ] Copy matched Google Docs

### PR DESCRIPTION
**Pull request by rosejbon**
Fixes #12 

## Description

- It is not very pretty but I have fitted in all of the words for privacy policy
-

@neontribe/contemplating-action

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -185,13 +185,20 @@ brandCopy key =
...
-            CopyText ""
+            CopyText "How secure is your data?"
 
         PrivacySectionSevenP ->
-            CopyText ""
+            CopyText "We have procedures in place to protect your data. These include password protection on all Hospice Hope computers and devices; password protected telephones for Hospice Hope, locked, secure cabinets for all paper documents with limited keyholders. Trustees, staff and key volunteers are all informed of their responsibility relating to data security and all staff, trustees and volunteers sign a confidentiality statement before they begin to work or volunteer for Hospice Hope. Hospice Hope only shares data with approved agencies e.g. HMRC re Gift Aid or Social Services in case of any safeguarding incident. However, if you think your data has been given to another organisation without authorisation, please contact our Administrator on 07935 800 658 or email on administrator@hospicehope.org.uk."
 
         PrivacySectionSevenList ->
-            CopyText ""
+            CopyList
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: I think this needs to be a `CopySection` otherwise it'll get bullet-ted.

**Review by katjam — COMMENTED** [review]
I think change the CopyList to CopySection and will be good to go.

See the Info Keys for example

File: src/Copy/BrandCopy.elm (hunk 1 above)
Comment (rosejbon) [reply src/Copy/BrandCopy.elm]: Ahhh, I didn't know you could do that

Comment (rosejbon) [reply src/Copy/BrandCopy.elm]: Made that change

**Review by Nikomus — APPROVED** [review]
Nice one!  📜©️

**Pull request merged**